azure-iot-device/azure/iot/device/provisioning/pipeline/pipeline_stages_provisioning.py

```diff
@@ -44,3 +58,388 @@ def _run_op(self, op):
 
         else:
             super(UseSecurityClientStage, self)._run_op(op)
+
+
+class CommonProvisioningStage(PipelineStage):
+    @pipeline_thread.runs_on_pipeline_thread
+    def _get_error(self, provisioning_op, prov_op_name, error):
```

Comment:
This also only seems to be used in one place, so why is it shared amongst many classes?

Reply:
the `get_error` method us used for both stages

---
In reply to: [364538162](https://github.com/Azure/azure-iot-sdk-python/pull/432#discussion_r364538162) [](ancestors = 364538162)